chapter1/configuration.md

```diff
@@ -1,4 +1,52 @@
-- Webportal login
-- navigators, options
-  (Screen shots)
+## Basic Configuration
 
+### Launch webportal
+In `appliance_console`, we need to check whether `ManageIQ Server` is up. If not then we can start it by entering digit `14` in the advanced setting.  After that when we hit IPv4 address which is given in `appliance_console` in web browser we can see the ManageIQ's Login page.
```

Comment:
You either emphasis the "ManageIQ Server" text or underline it rather than giving it a inline code as it is not a command.

Reply:
done